README.md

```diff
@@ -513,67 +513,116 @@ from events
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "CommitLockingTableFullAccess",
+            "Effect": "Allow",
+            "Action": "dynamodb:*",
+            "Resource": "arn:aws:dynamodb:<AWS_REGION>:<AWS_ACCOUNT_ID>:table/myGlueLockTable"
+        }
+    ]
+}
 ```
-The `warehouse` conf must be provided, but it's overwritten by the adapter `location` in your profile or `custom_location` in model configuration.
-
-As it is now, due to some dbt internal, the iceberg catalog used internally has a hardcoded name `iceberg_catalog`.
-
-
-### Table materialization
-Using table materialization, as for the other formats, the table is dropped and then recreated.
-An example of config is:
+- To add `file_format: Iceberg` in your table configuration
+- To add a connections in your profile : `connections: name_of_your_iceberg_connector` (The adapter is compatible with the Iceberg Connector from AWS Marketplace with Glue 3.0 as Fulfillment option and 0.12.0-2 (Feb 14, 2022) as Software version)
+- To add the following config in your Interactive Session Config (in your profile):  
+  ```--conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions 
+    --conf spark.serializer=org.apache.spark.serializer.KryoSerializer 
+    --conf spark.sql.warehouse.dir=s3://<your-bucket-name>/<your-prefix> 
+    --conf spark.sql.catalog.glue_catalog=org.apache.iceberg.spark.SparkCatalog 
+    --conf spark.sql.catalog.glue_catalog.warehouse=warehouse_path 
+    --conf spark.sql.catalog.glue_catalog.catalog-impl=org.apache.iceberg.aws.glue.GlueCatalog 
+    --conf spark.sql.catalog.glue_catalog.io-impl=org.apache.iceberg.aws.s3.S3FileIO 
+    --conf spark.sql.catalog.glue_catalog.lock-impl=org.apache.iceberg.aws.glue.DynamoLockManager 
+    --conf spark.sql.catalog.glue_catalog.lock.table=myGlueLockTable  
+    --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions```
+
+dbt will run an [atomic `merge` statement](https://iceberg.apache.org/docs/latest/spark-writes/) which looks nearly identical to the default merge behavior on Snowflake and BigQuery. You need to provide a `unique_key` to perform merge operation otherwise it will fail. This key is to provide in a Python list format and can contains multiple column name to create a composite unique_key. 
+
+##### Notes
+- When using a custom_location in Iceberg, avoid to use final trailing slash. Adding a final trailing slash lead to an un-proper handling of the location, and issues when reading the data from query engines like Trino. The issue should be fixed for Iceberg version > 0.13. Related Github issue can be find [here](https://github.com/apache/iceberg/issues/4582).
```

Comment:
Is there a easy way to force to remove the trailing slash programatically to avoid errors?

Reply:
I left this part about the trailing slash problem as it was before my PR because I haven't worked on it. It may be worth looking to see if the problem still exists, otherwise we can remove it from the documentation.